pkg/database/database.go

```diff
@@ -282,13 +298,12 @@ func (db *Database) WriteUrls(ctx context.Context, k8sObj *unstructured.Unstruct
 	}
 
 	for _, log := range logs {
-		_, execErr := tx.ExecContext(
-			ctx,
-			db.info.GetWriteUrlSQL(),
-			k8sObj.GetUID(),
+		q, qargs := db.Inserter.UrlInserter(
```

Comment:
I think it makes more sense this way, given we don't want to shadow the parent `query` and `args`.
```suggestion
		logQuery, logArgs := db.Inserter.UrlInserter(
```